pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[tool.poetry]
+name = "dump-env"
+version = "0.2.2"
+description = "A utility tool to create .env files"
+
+license = "MIT"
+
+authors = [
+  "Nikita Sobolev <mail@sobolevn.me>"
+]
+
+readme = "README.md"  # Markdown files are supported
+
+repository = "https://github.com/sobolevn/dump-env"
+homepage = "https://github.com/sobolevn/dump-env"
+
+keywords = [
+  "dump-env",
+  "flake8",
+  "plugin",
+  "linting",
+  "wemake.services"
+
+]
+
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "Environment :: Console",
+  "Framework :: Flake8",
+  "Intended Audience :: Developers",
+  "Operating System :: OS Independent",
+  "Topic :: Software Development :: Libraries :: Python Modules",
+  "Topic :: Software Development :: Quality Assurance",
+]
+
+[tool.poetry.plugins."flake8.extension"]
+Z = "wemake_python_styleguide.checker:Checker"
+
+[tool.poetry.dependencies]
+python = "^3.6 || ^3.7"
+flake8 = "^3.5"
+
+# This is a fix for issue-118
+pycodestyle = "==2.3.1"
+
+# delegator.py - didn't find this one
```

Comment:
You are not supposed to do this by hand. Run `poetry add --dev DEPENDENCY_NAME` for each dev-dependency we have.

It will generate the correct file with the correct versions.
So, please remove all the dev dependencies manually, then add them via `poetry add`.